distribution/src/bin/elasticsearch

```diff
@@ -45,6 +60,14 @@ then
   fi
 fi
 
+if [[ $ENROLL_TO_CLUSTER = true ]]; then
+  ES_MAIN_CLASS=org.elasticsearch.xpack.security.cli.EnrollNodeToCluster \
```

Comment:
I think we should just call `elasticsearch-enroll-node` here?

Reply:
Yes, we can I guess. The original plan was that the CLI tool is temporary for alpha2 and that we'd remove it but we found a proper use case for it and it stays. I didn't think about changing back to calling the tool but I agree it's more concise.